cargo-shuttle/tests/integration/run.rs

```diff
@@ -396,3 +301,100 @@ async fn rocket_hello_world_with_router_ip() {
 
     assert_eq!(request_text, "Hello, world!");
 }
+
+// These examples use a shared Postgres/Mongo. Thus local runs should create a docker containers.
+mod needs_docker {
```

Comment:
nit: rename to `docker_tests`.

Reply:
We are not testing docker, we are running tests that need docker to run ;)